dashboard/app/models/levels/level.rb

```diff
@@ -656,6 +656,9 @@ def clone_with_suffix(new_suffix, editor_experiment: nil)
 
     return Level.find_by_name(new_name) if Level.find_by_name(new_name)
 
+    # explicitly don't clone blockly levels (will cause a validation failure on non-unique level_num)
+    return self if key.start_with?('blockly:')
```

Comment:
nit: it would be slightly safer to put this before we check for new_name... in the extremely unlikely chance that we end up with a level named blockly_2022 😛  but also in the murkier case which has come up on levelbuilder where some of the blockly levels do actually have names other than "blockly".

the fix looks right to me. it would be nice to still find a way to verify it, but if that proves difficult then a speculative fix could also be ok. in that case it could be enough to convince yourself that you haven't made anything worse locally, even if it doesn't prove things will be fixed on levelbuilder.